include/MiniLua/package.hpp

```diff
@@ -0,0 +1,36 @@
+#ifndef MINILUA_PACKAGE
+#define MINILUA_PACKAGE
+#endif
```

Comment:
the endif should probably be moved to the end of the file

Reply:
resolved in #149